hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/ha/SequenceIdGenerator.java

```diff
@@ -79,7 +79,8 @@ static class Batch {
 
   private final Lock lock;
   private final long batchSize;
-  private final StateManager stateManager;
+  private StateManager stateManager;
+  private final SCMHAManager scmhaManager;
```

Comment:
This looks unused as a field, can we keep it local to the constructor?

Reply:
Right, it can be removed.